File: DAY 32/lgcl1.c
#include <stdio.h>

void main()
{
    int x = 40 ,y =35 , z= 20 , w = 10;
    int a = x * y / z-w;
    int b = x * y / (z-w);
    printf("%d ",a);
    printf("%d",b);
}
//output 
60  140

File: DAY 42/lgcl1.java
public class test 
{		
	public static void main(String[] args) 
	{	        		   
           B b = new D();
           b.show();
    }
}

class B
{
  public static void show()
  {
	  System.out.println("Base :: show() called");
  }
}

class D extends B
{
   public static void show()
   {
	   System.out.println("Derived :: show() called");
   }
}

//output "Base :: show() called" static method cant be overridden.
